pre-docs/inventory_version2.md

```diff
@@ -352,7 +353,76 @@ google_compute_instance.app.1:
   zone = us-west1-a
 ```
 
-#### Prompt plugin
+### AWS EC2
+
+The AWS EC2 plugin supports looking up running AWS EC2 instances in a specified region. It supports several fields:
+
+- `profile`: The [named profile](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-profiles.html) to use when loading from AWS `config` and `credentials` files. (optional, defaults to `default`)
+- `region`: The region to look up EC2 instances from. (optional)
```

Comment:
I think this might actually be required by the `aws-sdk`?

Reply:
Setting a region is definitely required, but it can be through an environment variable or through the credentials file. Do we want to require it being set in the inventory file only?